tests/unit/AddEditOrgPerson.spec.ts

```diff
@@ -128,9 +128,9 @@ const emptyPerson = { ...EmptyOrgPerson }
  * @returns the value of the last named event for the wrapper.
  */
 function getLastEvent (wrapper: Wrapper<AddEditOrgPerson>, name: string): any {
-  const eventsList: Array<any> = wrapper.emitted(name)
-  const events: Array<any> = eventsList[eventsList.length - 1]
-  return events[0]
+  const eventsList = wrapper.emitted(name)
+  const events = eventsList && eventsList[eventsList.length - 1]
+  return events && events[0]
```

Comment:
Previously, an error with an emitted event could report an error on line 132. Now, this function is safe, and the error will be reported in the unit test itself (ie, with the correct line number displayed).